ci/test_wheel_cuproj.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+mkdir -p ./dist
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="cuproj_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+
+# Install additional dependencies
+apt update
+DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libgdal-dev
+python -m pip install --no-binary fiona 'fiona>=1.8.19,<1.9'
+python -m pip install --no-binary cupy 'cupy>=12.0.0'
```

Comment:
Why use a source distribution when you could use a prebuilt wheel?

cupy-cuda11x / cupy-cuda12x is what I would recommend here if you must preinstall cupy for some reason. However, cupy-cuda* should be specified as a dependency of cuproj already so I am surprised that you need to preinstall cupy at all.

Reply:
Brad, I'm a total noob to the Python packaging game. I don't know the difference -- how can you tell that's a source distribution? I'm just seeing examples that are already there and modifying them until they work. Code suggestions welcome.

Reply:
I tried to use dependencies.yaml to get cupy installed but it didn't seem to work for wheel tests.

Reply:
No worries. This package describes itself as a source distribution on PyPI: https://pypi.org/project/cupy/

The packages you want for cupy are called cupy-cuda11x / cupy-cuda12x, and this is further described in the docs: https://docs.cupy.dev/en/stable/install.html#installing-cupy-from-pypi

My suggestion would be to remove this line. If CI fails, I can help identify the root cause and fix.

```suggestion
```

Reply:
You were correct, it passed!